bancorprotocol/carbon-app
# Trade Count

fix #1336

- Add Trade count on strategy card
- Remove everything related to roi

<img width="313" alt="image" src="https://github.com/user-attachments/assets/6f1fa7ce-d245-4c5c-8e9d-b9ff21b51d64">
<img width="246" alt="image" src="https://github.com/user-attachments/assets/ff4b9f40-9b69-497b-811f-3502f8e023d0">

What to test :
- Go to explorer page with pair ETH/USDC
- Strategies should have a trade count
- Click Filter/Sort
- Default sort should be trade, which sort by descending order
